packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -511,17 +511,23 @@ def _download_bundle_from_master(dcos_api_session, master_index):
     expected_common_files = ['dmesg-0.output.gz', 'opt/mesosphere/active.buildinfo.full.json.gz',
                              'opt/mesosphere/etc/dcos-version.json.gz', 'opt/mesosphere/etc/expanded.config.json.gz',
                              'opt/mesosphere/etc/user.config.yaml.gz', 'dcos-diagnostics-health.json',
-                             'var/lib/dcos/cluster-id.gz']
+                             'var/lib/dcos/cluster-id.gz', 'ps_aux_ww.gz', '/proc/cmdline', '/proc/cpuinfo',
```

Comment:
The files currently called `'/proc/cpuinfo'` etc. should be `'proc/cpuinfo.gz'`, etc.

Reply:
Fixed. Test passed because `/proc/cpuinfo` is existing file on the system